src/components/ModelsDataCard/modelsDataCard.module.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../assets/variables";
+
+$green: #74db77;
+$red: #c71c1c;
+
+.active {
+  color: $green;
+  font: $text-error-message-semibold;
+  font-size: 18px;
+}
+
+.disabled {
```

Comment:
Why not add some class with font and font size styles, and have only different colors in .active and .disabled classes